kpops/cli/main.py

```diff
@@ -205,6 +205,17 @@ def generate(
         None, help="Path to file the yaml pipeline should be saved to"
     ),
     verbose: bool = typer.Option(False, help="Enable verbose printing"),
+    template: bool = typer.Option(False, help="Run helm template"),
+    steps: Optional[str] = PIPELINE_STEPS,
+    api_versions: str = typer.Option(
```

Comment:
I believe this should be a `list[str]`. wdyt?

Reply:
If you are talking about `api_versions`, yes it is a `list[str]` in the real helm command, but we agreed with @raminqaf that for now I can leave it at `str` as it has enough functionality and at a later point change it into a list.

Reply:
ok, wouldn't it make sense to rename the argument to `api_version` (singular) then? cc @raminqaf

Reply:
Yeah good point let's keep it singular

Reply:
Done